src/test/scala/fr/acinq/bitcoin/BtcAmountSpec.scala

```diff
@@ -68,9 +68,9 @@ class BtcAmountSpec extends FunSuite {
   }
 
   test("negate amount") {
-    assert(Satoshi(-20) === -Satoshi(20))
-    assert(MilliBtc(-1.5) === -MilliBtc(1.5))
-    assert(Btc(-2.5) === -Btc(2.5))
+    assert(Satoshi(-20) === Satoshi(20).unary_-())
+    assert(MilliBtc(-1.5) === MilliBtc(1.5).unary_-())
+    assert(Btc(-2.5) === Btc(2.5).unary_-())
```

Comment:
I'm not very happy with that but it seems like Scala 2.13.3 deprecated applying unary operators without explicit arguments :/
TBH this isn't something we use much for bitcoin amounts.

Reply:
We can keep the code as it was if we just remove the `()` from the definition of `unary_-()` i.e in classes Satoshi, MilliBtc and Btc we would change:
```scala
def unary_-() = Satoshi(-underlying)
```
to
```scala
 def unary_- = Satoshi(-underlying)
```

Reply:
Good catch! Fixed in https://github.com/ACINQ/bitcoin-lib/pull/44/commits/7d3622c628c7b1ff51ccf69d7aaae6ce4d664e67